packages/account-v2/src/components/base/WalletDropdown/WalletDropdown.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { useCombobox } from 'downshift';
+import { StandaloneChevronDownRegularIcon } from '@deriv/quill-icons';
+import { TGenericSizes } from '../types';
+import { reactNodeToString } from '../utils';
+import { WalletText } from '../WalletText';
```

Comment:
I think you can use Text and other components from `deriv/quill-design` instead of making your own

Reply:
@amina-deriv  can u comment on this

Reply:
@likhith-deriv use `#architecture_frontend_deriv_app` channel to communicate with the team in charge of deriv/ui. I don't recommend creating duplicated base components!

Reply:
This is a dummy component until we get Dropdown from deriv-com/ui-library.